package/system/uci/Makefile

```diff
@@ -22,7 +22,6 @@ PKG_LICENSE_FILES:=
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@nbd.name>
 
-PKG_BUILD_PARALLEL:=0
```

Comment:
I'd just leave this line for now and move on with the rest.

Reply:
It's not needed as it's being built with Ninja now. I've tested multiple clean compiles and it has not failed even once.